frontends/concrete-python/examples/pir/PIR.ipynb

```diff
@@ -0,0 +1,577 @@
+    "                f\"FHE computation done in {(time_end - time_begin) * 1000:.1f} milliseconds -- database is {database_length} (2**{log_database_length}) elements of {database_output_bits} bits\"\n",
+    "            )\n",
+    "\n",
+    "        # Decrypt the result on the client side\n",
+    "        y = circuit.decrypt(encrypted_y)\n",
+    "\n",
+    "        # And check the computations worked fine\n",
+    "        assert y == database[random_index]\n",
+    "\n",
+    "    return times\n",
+    "\n",
+    "\n",
+    "_ = test_encrypted_queries(database, circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9716a15-0797-4270-b08a-7d1db97323e2",
+   "metadata": {},
+   "source": [
+    "## Performances\n",
+    "\n",
+    "Now, obviously, we can do it for much larger databases. Let's see the performance!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "3a573631-0bdf-4969-85c1-c5da9afcc2bf",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "For a database of 2** 4 elements of  8 bits, average execution time is 1.1 milliseconds\n",
+      "For a database of 2** 4 elements of 16 bits, average execution time is 1.2 milliseconds\n",
+      "For a database of 2** 8 elements of  8 bits, average execution time is 3.8 milliseconds\n",
+      "For a database of 2** 8 elements of 16 bits, average execution time is 6.1 milliseconds\n",
+      "For a database of 2** 9 elements of  8 bits, average execution time is 8.2 milliseconds\n",
+      "For a database of 2** 9 elements of 16 bits, average execution time is 9.4 milliseconds\n",
+      "For a database of 2**10 elements of  4 bits, average execution time is 7.5 milliseconds\n",
+      "For a database of 2**10 elements of  8 bits, average execution time is 9.6 milliseconds\n",
+      "For a database of 2**12 elements of  4 bits, average execution time is 32.1 milliseconds\n",
+      "For a database of 2**12 elements of  8 bits, average execution time is 45.3 milliseconds\n",
+      "For a database of 2**14 elements of  4 bits, average execution time is 169.0 milliseconds\n",
+      "For a database of 2**14 elements of  8 bits, average execution time is 195.8 milliseconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "how_many_tests = 10\n",
+    "\n",
+    "sample_list = [(4, 8), (4, 16), (8, 8), (8, 16), (9, 8), (9, 16), (10, 4), (10, 8), (12, 4), (12, 8), (14, 4), (14, 8)]\n",
+    "timings_dic = {}\n",
+    "\n",
+    "for database_input_bits, database_output_bits in sample_list:\n",
+    "\n",
+    "    # Take a random database of expected size and output_bits\n",
+    "    database_length = 2**database_input_bits\n",
+    "    database = np.array(\n",
+    "        [np.random.randint(2**database_output_bits) for _ in range(database_length)]\n",
+    "    )\n",
+    "\n",
+    "    circuit = compile_function(database, show_mlir=False, show_graph=False)\n",
```

Comment:
so maybe just remove kwargs as it's default one

Reply:
I would keep them just for the user to know they exist!